release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2143,6 +2143,15 @@ $ oc get etcd '-o=jsonpath={range .items[0].status.conditions[?(@.type=="NodeIns
 * The web console downloads deployment will fail on nodes configured with `ipv6.disable=1`.
  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1795325[*BZ1795325*])
 
+* An issue in Topology Manager could result in NUMA resources not being aligned to the same NUMA node if Guaranteed QoS pods are created simultaneously on the same node.
+As a result, the resources requested in the pod spec might not be NUMA aligned. This issue is fixed in {product-title} 4.5.
++
+To avoid this issue in {product-title} 4.4, do not spin up multiple pods with a Guaranteed QoS on a node simultaneously.
++
+If you encounter this issue, delete and then recreate the pod.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1834979[*BZ#1834979*])
```

Comment:
We typically try to keep the BZ link right after the last sentence (following the period). There are several cases where the BZ link was added on the next line, but those were only after steps or a command.

Reply:
outdated